src/logging/interface.js

```diff
@@ -5,13 +5,19 @@
  * NewRelicLoggingService that sends messages to NewRelic that can be seen in NewRelic Browser and
  * NewRelic Insights. When in development mode, all messages will instead be sent to the console.
  *
- * When you use `logError`, your errors will appear under "JS errors"
- * for your Browser application.
+ * When you use `logError`, your errors will be checked to see if they're ignored *or* not.
```

Comment:
Could we have docs just in NewRelicLoggingService.js to avoid duplication, and reference them from here? Also, note that these docs are partially specific to the New Relic implementation.

Reply:
Fine with me - will do.